### DISCUSSION TITLE
Interpreting the result of link_prediction explainer?

### DISCUSSION THREAD

**Discussion by kal-purush** [pyg-team/pytorch_geometric#6716] [Q&A]
How can I interpret the result of link_prediction explainer? 
I run the example code from the git repo and this the output

```
tensor([0.1473, 0.1484, 0.1455,  ..., 0.1462, 0.1458, 0.1503])
tensor([[0.1732, 0.1679, 0.1755,  ..., 0.1613, 0.1687, 0.1801],
        [0.1777, 0.1700, 0.1726,  ..., 0.1658, 0.1579, 0.1625],
        [0.1720, 0.1789, 0.1607,  ..., 0.1541, 0.1602, 0.1825],
        ...,
        [0.1782, 0.1908, 0.1695,  ..., 0.1556, 0.1664, 0.1493],
        [0.1819, 0.1635, 0.1719,  ..., 0.1755, 0.1692, 0.1742],
        [0.1662, 0.1824, 0.1678,  ..., 0.1532, 0.1635, 0.1740]])
```

But I am not sure how to interpret this? I want to understand how a certain edge was predicted i.e., which features or edges were used to predict a particular edge.

**Comment by wsad1** [comment]
Which example from `pyG` did you run, was it [this](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gnn_explainer_link_pred.py).
If so the first tensor is the `edge_mask` indicating which edges are important, and the second is the feature importance mask for each node.

**Reply by rusty1s** [reply]
Discussed on Slack as well. I think there are still some issues with `GNNExplainer` on link-level tasks since we never clear feature importance/edge importance for edges/nodes never visited. I imagine a simple way to fix this would be to compare `node_mask` before and after `GNNExplainer` runs, and simply zero out any score that has not changed. WDYT?

**Comment by rusty1s** [comment]
I fixed this in https://github.com/pyg-team/pytorch_geometric/pull/6787.
